packages/babel-helper-create-class-features-plugin/src/fields.js

```diff
@@ -429,7 +471,47 @@ function buildPublicFieldInitSpec(ref, prop, state) {
 }
 
 function buildPrivateStaticMethodInitLoose(ref, prop, state, privateNamesMap) {
-  const { id, methodId } = privateNamesMap.get(prop.node.key.id.name);
+  const privateName = privateNamesMap.get(prop.node.key.id.name);
+  const { id, methodId, getId, setId, initAdded } = privateName;
+  if (initAdded) return;
+
+  if (getId || setId) {
+    privateNamesMap.set(prop.node.key.id.name, {
+      ...privateName,
+      initAdded: true,
+    });
+
+    if (getId && setId) {
+      return template.statement.ast`
+        Object.defineProperty(${ref}, ${id}, {
+          // configurable is false by default
+          // enumerable is false by default
+          // writable is false by default
+          get: ${getId.name},
+          set: ${setId.name}
```

Comment:
Ditto reuse.

Reply:
That works in [this case](https://github.com/babel/babel/pull/10217/files/14f480de578c5f6220a8f8d62a31dd3fe1833ce2#r305591019), but setting `null` as a getter/setter is a `TypeError`.

Reply:
Ah, yah. We should be able to use `undefined` in all cases instead.

Reply:
Good point, updated.